SR2022/2022-03-09-status-update-still-silent-teams.md

```diff
@@ -0,0 +1,20 @@
+---
+to: Student Robotics 2022 teams we haven't heard from in a while (chase 2)
+subject: Let us know how your robot is progressing
+---
+
+Hi {team-supervisor-name},
+
+We haven't heard from your team in a while and wanted to check how you're doing?
```

Comment:
Would be great to have a different intro to the one in https://github.com/srobo/team-emails/pull/103, so it's clear to the reader that this is a different email. A different subject line would probably also be a good idea.